hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/om/om.tsx

```diff
@@ -567,7 +567,7 @@ export class Om extends React.Component<Record<string, object>, IOmdbInsightsSta
     const { request, controller } = AxiosGetHelper(openKeysEndpoint, cancelOpenKeysSignal)
     cancelOpenKeysSignal = controller
     request.then(openKeysResponse => {
-      const openKeys = openKeysResponse && openKeysResponse.data;
+      const openKeys = openKeysResponse?.data;
```

Comment:
This will short-circuit to undefined in case data is not defined.
Please add a fallback value to this
```suggestion
      const openKeys = openKeysResponse?.data ?? {"fso": []};
```

Reply:
@devabhishekpal 
Done Changes in latest commit 5ae3e461eba37df92fbb93f5bb64f5439b36d3b8